lib/WebService/Async/Onfido.pm

```diff
@@ -64,7 +65,7 @@ my %FILE_MIME_TYPE_MAPPING = (
 
 sub configure {
     my ($self, %args) = @_;
-    for my $k (qw(token requests_per_minute base_uri)) {
+    for my $k (qw(token requests_per_interval base_uri rate_interval)) {
```

Comment:
Why are you breaking public interface?

Reply:
I think the size of  `minute` is too big. 
Yes you are right. let me change it back.
But that will make test more difficulty (sleep 1 minute ?)